PackagingProjects/CSharpPluginPayload/CSharpPluginPayload.csproj

```diff
@@ -29,12 +29,14 @@
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\</OutputPath>
     <DefineConstants>TRACE</DefineConstants>
+    <NoWarn>CS2008</NoWarn>
   </PropertyGroup>
   <PropertyGroup>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Miscellaneous: added this to this project and two others to fix the compiler warnings about different versions of the NewtonSoft being referenced. We are only referencing v6, but for some reason v4 is also being pulled in as a dependency.